packages/server/graphql/mutations/helpers/notifications/getSummaryText.ts

```diff
@@ -5,10 +5,17 @@ import {isMeetingAction} from '../../../../database/types/MeetingAction'
 import {isMeetingPoker} from '../../../../database/types/MeetingPoker'
 import {isMeetingRetrospective} from '../../../../database/types/MeetingRetrospective'
 import {isMeetingTeamPrompt} from '../../../../database/types/MeetingTeamPrompt'
+import sendToSentry from '../../../../utils/sendToSentry'
 
 const getSummaryText = (meeting: Meeting) => {
   if (isMeetingRetrospective(meeting)) {
     const {commentCount = 0, reflectionCount = 0, topicCount = 0, taskCount = 0} = meeting
+    const hasNonZeroStat = commentCount || reflectionCount || topicCount || taskCount
+    if (!hasNonZeroStat && meeting.summary) {
```

Comment:
Did they have the AI summary enabled? From the slack screenshot I don't see a AI summary section displayed.